### PR TITLE
A Vote for Governance Rephrase

### LINKED ISSUE #124: Update community governance description
subtask of #118 

With consensus in #118 we'd like to formalize the governance and make it no longer a draft. This issue is a subtask tracing all contents updates. So far, it seems containing

* TOC description
* Team description

Also we should remove all stale contents including `committee/`, `sig/`, and `wg/`.

cc @luzizhuo @gingerkidney 

subtasks

* [x] #131

### PR DESCRIPTION
**Pull request by tisonkun**
closes #124 

Description structure 

* Roles and Responsibilities
  * Steering Committee
  * Teams
    * Reviewers
    * Committers
    * Maintainers
  * Infrastructure
* Decision Making
  * Voting
  * Approvals
  * Vetoes
  * Actions

Also several content should be cleared after this pull request is accepted.

* `committee/`
* `sig/`
* `wg/`
* `commnuity-membership.md`
* `commnuity-membership-zh_CN.md`
* `GOVERNANCE-zh_CN.md`

We may include more guideline later but the governance is the checkpoints.

Render version: https://github.com/tikv/community/blob/6e1d196f64b2a5da7d6c6e8a821e3e085370943d/GOVERNANCE.md

### CODE REVIEW

**Comment by tisonkun** [comment]
/cc @BusyJay @winkyao @sunxiaoguang @zhangjinpeng1987

in cc @siddontang @lidaobing @fredchenbj

**Review by sunxiaoguang — APPROVED** [review]
LTGM

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

File: GOVERNANCE-stage.md
Comment (BusyJay) [inline GOVERNANCE-stage.md:62]: What if there is only one maintainer? I think it should be able to remove a maintainer by active committee members.

Comment (tisonkun) [reply GOVERNANCE-stage.md:62]: You are right the the committee can dominate the teams if necessary though we'd better avoid actually doing that. Let me try to explicitly describe this case.

File: GOVERNANCE-stage.md
Comment (winkyao) [inline GOVERNANCE-stage.md:58]: >The maintainers consist group of active committers that moderate the discussion, manage the project release, and proposes new committers or maintainers. Potential candidates are usually proposed via an internal discussion among maintainers, followed by a consensus approval, i.e. a concrete number of approvals, and no vetoes. Any veto must be accompanied by reasoning. Maintainers should serve the community by upholding the community practices and guidelines TiDB a better community for everyone. Maintainers should nominate new reviewer, committers and maintainers, and should also strive to only nominate new candidates outside of their own organization.

TiDB -> TiKV

Comment (tisonkun) [reply GOVERNANCE-stage.md]: Updated.

Comment (tisonkun) [reply GOVERNANCE-stage.md]: PTAL

File: GOVERNANCE-stage.md
Comment (tisonkun) [reply GOVERNANCE-stage.md:62]: Updated 1c2668a1928435477ff28512627ac4a4a55a6fee . Though the better way I think is grow the maintainer group at the case. Because diversity is important of a community and in community we trust instead of setting up policies.

File: GOVERNANCE-stage.md
Comment (winkyao) [inline GOVERNANCE-stage.md:127]: New Committer and New Maintainer should have longer "Minimum Length"? I think it should be more than 6 days.

Comment (winkyao) [reply GOVERNANCE-stage.md:127]: and the New Committee Member too

Comment (tisonkun) [reply GOVERNANCE-stage.md]: updated as 2f011a161e8f577a772012ba9937e08a03d83f12

Comment (tisonkun) [reply GOVERNANCE-stage.md]: @BusyJay PTAL. I'm unsure whether it is a reasonable length for new committer.

**Review by winkyao — COMMENTED** [review]
LGTM

**Review by BusyJay — COMMENTED** [review]
LGTM

**Comment by tisonkun** [comment]
/hold

Thanks for your review! Hold because after this pull request is ready to merge, I will move the `-stage` file to a formal file, and also remove the filed proposed to drop.

**Review by zhangjinpeng1987 — APPROVED** [review]
LGTM

**Comment by tisonkun** [comment]
@BusyJay I've renamed the `stage` file to formal file. Please give it a final check. I'll create another PR to clean up previous stuff.

**Review by BusyJay — APPROVED** [review]

**Comment by tisonkun** [comment]
@BusyJay @sunxiaoguang @winkyao @zhangjinpeng1987 Thanks for your review.

According to the governance content we'd better keep this pull request open by the end of 7.3 since it requires 6 days ad a minimal length. I'll ping you then.

By the same time you can take a look at #135 which is a follow up of this pull request. It has nothing to do with decision making so we can merge it right after this PR gets merged. Or we can even integrate #135 into #133 . Look forward to your suggestion.

**Comment by tisonkun** [comment]
@BusyJay I think we have over the minimal length and receive 4 approvals. I have updated the result and it is now mergeable from my perspective. Would you mind to trigger the merge phase?

BTW, the clean step at #135 is waiting for review.

**Pull request merged**
